api/ContactAddress.json

```
@@ -0,0 +1,53 @@
+{
+  "api": {
+    "ContactAddress": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "13",
+            "version_removed": "23"
+          },
+          "firefox_android": {
+            "version_added": "14",
+            "version_removed": "23"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": true
```

```suggestion
            "version_added": "60"
```

Mapped from Chrome, but also tested in Opera 61 to confirm it's not disabled.